### PR TITLE
Add TypeStatement for Events

### PR DESCRIPTION
**Pull request by joecummings**

### CODE REVIEW

File: params/claims_pipeline.params [hunk 1]
```diff
@@ -43,10 +43,6 @@ aif:
     python_file: "%project_root%/cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py"
     aif_output_dir: "%experiments_root%/aif_output"
 
-aif:
```
Comment (elizlee) [inline params/claims_pipeline.params:46]: Why were these removed?

**Review by elizlee — COMMENTED** [review]
One question

File: params/claims_pipeline.params (hunk 1 above)
Comment (joecummings) [reply params/claims_pipeline.params:46]: it was duplicated: https://github.com/joecummings/cdse-covid/blob/46627cb44ecfe637b7fb6f8f306ddfa857853f57/params/claims_pipeline.params#L46

Comment (elizlee) [reply params/claims_pipeline.params:46]: Huh, okay then!
